docs/component-text.md

```diff
@@ -0,0 +1,32 @@
+# Component Text
+
+React Native Base Component Text
+
+## Text Props
+ this component extends from React Native Props.
+with some default Type Text
+
+## Usage
+
+```js
+import {BaseText} from 'rn-base-component';
+
+<BaseText> BaseText </BaseText>
```

Comment:
Why the document and the implementation are different?

Reply:
I forgot that we have the documents, let me update

Reply:
updated